proto/rzd_service.proto

```
@@ -0,0 +1,146 @@
+syntax = "proto3";
+
+package rzd;
```

buf (1.47.2)

Files with package "rzd" must be within a directory "rzd" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)